.circleci/config.yml

```diff
@@ -246,7 +246,7 @@ jobs:
       - run: brew install cmake
       - <<: *clang-debug
     macos:
-      xcode: "11.2.1"
+      xcode: "14.2.0"
```

Comment:
CircleCI no longer support Xcode 11, neither do Apple. Not feasible to test anymore.